.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+- [ ] Add [CHANGELOG](https://github.com/timescale/timescaledb/blob/master/CHANGELOG.md) updates
+- [ ] Needs [documentation](https://github.com/timescale/docs) updates
+
+Fixes #<issue number>.
+
+---
```

Comment:
Can we somehow show this information without it being part of the template body.

Reply:
Without it there essentially no template. Yes, it is mentioned below as well, but it can easily be removed of course.